docs/blog/20230326-Daeraxa-Survey1-Results.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Survey Results: How did you hear about us?"
+author: Daeraxa
+date: 2023-03-26
+category:
+  - survey
+tag:
+  - community
+  - socials
+---
+
+Here we have the results and analysis of our first community survey (and an infographic for those who like that kind of thing).
+
+<!-- more -->
+
+A big thank you to everyone who participated in our recent social media survey. This really helped answer some questions that we had in order to be able to make sure we were reaching as much of the community as possible. We were honestly surprised at some of the results and we will definitely be taking lots of it on board.
+
+First of all here is an infographic we put together on the data that was provided.
+
+<!-- markdownlint-disable -->
+
+<div class="image-preview">
+  <img src="./assets/survey1-infographic1.png" />
+  <img src="./assets/survey1-infographic2.png" />
+</div>
+
+<style>
+  .image-preview {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .image-preview > img {
+     box-sizing: border-box;
+     width: 33.3% !important;
+     padding: 9px;
+     border-radius: 16px;
+  }
+
+  .image-preview > img:hover {
+    cursor: zoom-in;
+  }
+
+  @media (max-width: 719px){
+    .image-preview > img {
+      width: 50% !important;
+    }
+  }
+
+  @media (max-width: 419px){
+    .image-preview > img {
+      width: 100% !important;
+    }
+  }
+</style>
+
+<!-- markdownlint-restore -->
+
+And now onto the analysis.
+
+## How did you hear about us?
+
+The first question on the survey was "How did you hear about us"? The idea with this one was to see where people had heard about Pulsar from to see if this aligned with some of our earlier efforts to publicise the project.
+
+By far the biggest result here was simply people searching the web for an Atom replacement, I don't think there were any shockers here as this is how many of us got involved in the first place.
+
+A slightly more surprising result was the number of people who found out via Reddit, mostly [/r/atom](https://www.reddit.com/r/atom) but also from a number of programming or other related subreddits.
+
+We also had a lot of people come from seeing a YouTube video on the project and it seems a considerable number of those were from the [DistroTube channel](https://www.youtube.com/@DistroTube) (which we mentioned in a [previous blog](https://pulsar-edit.dev/blog/20221208-Daeraxa-DistroTubeVideo.html)). It really goes a long way to show the effect that community content creators in this space can have on a project and its popularity.
+
+We had a couple of other interesting results here such as word of mouth from colleagues, various web articles, alternative software sites and even the Atom Wikipedia page (although Pulsar seems to have been removed from it now).
```

Comment:
Maybe it was temporary gone when writing this blog, but the reference to Pulsar on Atom's Wikipedia page it there new. Its the last line under History.

Reply:
You're right, someone added it back, thanks for the heads up!

Reply:
Thanks, I'll remove that again now